lib/trento/users.ex

```diff
@@ -15,6 +15,14 @@ defmodule Trento.Users do
 
   alias Trento.Users.User
 
+  @spec by_id(Number.t()) :: {:ok, User.t()} | {:error, :not_found}
```

Comment:
I would suggest `non_neg_integer()` as a more precise typespec. I am not sure/fully aware if there is a `Number.t()`  type. :thinking: